source/develop/release-management/features/network/networking-api-security-groups.html.md

```diff
@@ -0,0 +1,597 @@
+| direction            | string | Ingress or egress, which is the direction in which the security group rule is applied.                                                                           |
+| protocol (*)         | string | Check https://developer.openstack.org/api-ref/network/v2/#create-security-group-rule                                                                             |
+| ethertype (*)        | string | Check https://developer.openstack.org/api-ref/network/v2/#create-security-group-rule                                                                             |
+| port_range_max (*)   | int    | The maximum port number in the range that is matched by the security group rule.                                                                                 |
+| port_range_min (*)   | int    | The minimum port number in the range that is matched by the security group rule.                                                                                 |
+| security_group_id    | string | The security group ID to associate with this security group rule.                                                                                                |
+| remote_ip_prefix (*) | string | The remote IP prefix that is matched by this security group rule.                                                                                                |
+| description (*)      | string | A human-readable description for the resource. Default is an empty string.                                                                                       |
+
+(*) - optional attributes
+(**) - optional attributes that ***will not*** be implemented.
+
+### OVN ACL table
+
+An OVN ACL has the following attributes:
+
+| name      | type   | description |
+|-----------|--------|-------------|
+| priority  | int    | Range from 0 to 32,767. The smaller the value, the higher the priority                                                                                   |
+| direction | string | Either 'from-lport' or 'to-lport'.                                                                                                                       |
+| match     | string | The match criteria for the ACL. All the L3-L4 information (protocol, ether protocol, ports, ip prefix) is encoded here. Directly relates to OVS matches. |
+| action    | string | Either 'allow', 'allow-related', 'drop', or 'reject'. Currently, 'reject' behaves as 'drop'.                                                             |
+
+### Mapping the rules
+
+Mapping is a two way game: it is needed to translate *from* a networking
+API security group rule *to* an OVN ACL, ***and*** from an OVN ACL to a
+networking API security group rule.
+
+To achieve the latter, the information that is not **directly** encoded
```

Comment:
I would empahsize the "not" instead / with the "directly"

Reply:
I agree. 

Done